manifests/server/config.pp

```diff
@@ -140,7 +140,13 @@
       mode   => $::puppet::server_environments_mode,
     }
 
-    # make sure your site.pp exists (puppet #15106, foreman #1708)
+    # make sure your site.pp exists (puppet #15106, foreman #1708) and server_manifest_path too
+    file { $puppet::server_manifest_path:
+      ensure => directory,
+      owner  => $puppet::server_user,
+      group  => $puppet::server_group,
+      mode   => '0755',
+    } ->
```

Comment:
I'm sure some work related stuff sneaked:snake: in here and it's really about Puppet 2.7. ;)